### PR TITLE
SC Settings Tuning

### LINKED ISSUE #587: Enhance SC Settings
Provide better BD-Rate/Speed trade-offs for SC contents

### PR DESCRIPTION
**Pull request by hguermaz**
## Description

SC Settings Tuning

Closes #587

## Author

@hguermaz 

## Type of change

New feature/Tool

## Tests and performance
* BD-Rate to aom cpu0-2pass on the 2 SC clips:  0.187% gain
* Speed on a 4-core machine:  TBD
* Memory footprint on a 4-core machine: negligeable

### CODE REVIEW

File: Source/Lib/Common/Codec/EbDefinitions.h [hunk 1]
```diff
@@ -58,6 +58,13 @@ extern "C" {
 #define PREDICTIVE_ME                     1 // Perform ME search around MVP @ MD
 #define MD_STAGING                        1
 
+
+
```
Comment (andrey-norkin) [inline Source/Lib/Common/Codec/EbDefinitions.h]: That many empty lines should be removed

Comment (hguermaz) [reply Source/Lib/Common/Codec/EbDefinitions.h]: Done

**Comment by JackZhouVCD** [comment]
This PR introduced some coding artifacts in the context of M3 as described in #633. @okhlif @hguermaz can you look into this?

**Comment by EwoutH** [comment]
@hassount Could this PR get a _Q+_ and _new feature_ label, for historical reference?

**Pull request merged**